timesketch/lib/charts/manager.py

```diff
@@ -41,13 +41,14 @@ def get_chart(cls, chart_name):
             chart_name (unicode): name of the chart to retrieve.
 
         Returns:
-            Chart class object.
+            Chart class object or None if not existing.
         """
-        return cls._class_registry[chart_name.lower()]
+        chart_class = cls._class_registry.get(chart_name.lower())
+        return chart_class
```

Comment:
you can also just have this as 

```
return cls._class_registry.get(chart_name.lower())
```

and skip the variable assignment

Reply:
Done.